set-web/src/main/java/eu/europeana/set/web/service/impl/BaseUserSetServiceImpl.java

```diff
@@ -895,6 +898,7 @@ PersistentUserSet updateUserSetForUnpublish(PersistentUserSet userSet,
     }
     
     userSet.setVisibility(VisibilityTypes.PUBLIC.getJsonValue());
+    userSet.setIssued(null);
```

Comment:
this must be executed, on line 891 an exception must be thrown, so that the api returns 400